tests/integration/api/mutations/removeTag/removeTag.test.js

```diff
@@ -0,0 +1,91 @@
+import Factory from "/imports/test-utils/helpers/factory";
+import TestApp from "/imports/test-utils/helpers/TestApp";
+import { encodeShopOpaqueId } from "@reactioncommerce/reaction-graphql-xforms/shop";
+import { encodeTagOpaqueId } from "@reactioncommerce/reaction-graphql-xforms/tag";
+import RemoveTagMutation from "./RemoveTagMutation.graphql";
+import Logger from "@reactioncommerce/logger";
+
+let testApp;
+let shopId;
+let removeTag;
+let mockTagsAccount;
+let tagInput;
+let fakeTag;
+
+beforeAll(async () => {
+  testApp = new TestApp();
+  await testApp.start();
+  shopId = await testApp.insertPrimaryShop();
+
+  mockTagsAccount = Factory.Accounts.makeOne({
+    roles: {
+      [encodeShopOpaqueId(shopId)]: ["owner"]
+    }
+  });
+  await testApp.createUserAndAccount(mockTagsAccount);
+
+  removeTag = testApp.mutate(RemoveTagMutation);
+});
+
+afterAll(async () => {
+  await testApp.collections.Shops.deleteMany({});
+  await testApp.stop();
+});
+beforeEach(async () => {
+  fakeTag = Factory.Tag.makeOne({
+    _id: encodeTagOpaqueId(123),
```

Comment:
Should we just use `accountInternalId = "123"` (line 51) in here (instead of hardcoding the same constant?